testing/btest/bifs/load_path.zeek

```diff
@@ -0,0 +1,49 @@
+# @TEST-DOC: Test load_path() and demo conditional @load'ing.
+#
+# @TEST-EXEC: zeek -b %INPUT >out
+# @TEST-EXEC: btest-diff out
+# @TEST-EXEC: btest-diff .stderr
+
+
+@if ( load_path("pkg1") != "" )
+@load pkg1
+@endif
+
+@if ( load_path("pkg2") != "" )
+@load pkg2
+@endif
+
+@if ( load_path("pkg3") != "" )
+@load pkg3
+@endif
+
+function path_tail(r: string): string
+	{
+	if ( |r| == 0 )
+		return r;
+	local parts = split_string(r, /\//);
+	return join_string_vec(parts[-4:], "/");
+	}
+
+print "load_path base/protocols/conn", path_tail(load_path("base/protocols/conn"));
+print "load_path protocols/conn (empty expected, no __load__.zeek)", load_path("protocols/conn");
+print "load_path protocols/conn/vlan-logging", path_tail(load_path("protocols/conn/vlan-logging"));
+
+print "load_path pkg1", load_path("pkg1");
```

Comment:
Can you sprinkly in some tests which exercise relative loads (involving `./`, `../`, a mix of them)?

Reply:
The relative loads are dependent on the parsing state (which file is currently being parsed?) and not a great fit for the BiF. Though, that make me wonder if there's a better name, or the docs need to emphasize this.